src/Renderer/TaskLineRenderer.ts

```diff
@@ -13,7 +13,7 @@ import { StatusMenu } from '../ui/Menus/StatusMenu';
 import { TaskFieldRenderer } from './TaskFieldRenderer';
 
 /**
- * The function used to render a Markdown task line into an existing HTML element
+ * The function used to render a Markdown task line into an existing HTML element.
```

Comment:
😅 if you're wondering where this came from. git rebase --continue wouldn't work during an ongoing massive rebase because it kept saying no file changed. so I had to make a small edit to get it to continue.